<launch>

    <arg name="baud" value="57600"/>
    <arg name="endpoint"            default="opc.tcp://172.22.132.13:4840"  />
    <arg name="port"                default="/dev/ttyACM0"  />
    
    <node name="arduino_node" pkg="rosserial_arduino" type="serial_node.py" args="$(arg port) $(arg baud)"/>
    <node name="opcua_node" pkg="ros_opcua_impl_freeopcua" type="client_node">
    	<param name="endpoint" value="$(arg endpoint)"/>
    </node>
    <node name="gripper_node" pkg="screw_gripper" type="gripper_system" output="screen"/>
      
</launch>
